scenarios/throughput_stress.go

```diff
@@ -613,6 +631,68 @@ func (t *tpsExecutor) createNexusWaitForCancelAction() *Action {
 	}
 }
 
+func (t *tpsExecutor) createScheduleCreateAction(scheduleID string) *Action {
+	return &Action{
+		Variant: &Action_CreateSchedule{
+			CreateSchedule: &CreateScheduleAction{
+				ScheduleId: scheduleID,
+				Spec: &ScheduleSpec{
+					CronExpressions: []string{"* * * * *"},
+				},
+				Action: &ScheduleAction{
+					WorkflowId:   fmt.Sprintf("%s-wf", scheduleID),
+					WorkflowType: "kitchenSink",
+					TaskQueue:    fmt.Sprintf("omes-%s", t.runID),
+					Input: []*common.Payload{
+						ConvertToPayload(&WorkflowInput{
+							InitialActions: []*ActionSet{
+								NoOpSingleActivityActionSet(),
+							},
+						}),
+					},
+				},
+				Policies: &SchedulePolicies{
+					RemainingActions:    1,    // Only run once if triggered
+					TriggerImmediately: true, // Fire immediately when created
+				},
+			},
+		},
+	}
+}
+
+func (t *tpsExecutor) createScheduleDescribeAction(scheduleID string) *Action {
```

Comment:
I think it'd be better to inline these 5 helpers as I think that's clearer to read and navigate.